examples/standard_interfaces/README.md

```diff
@@ -43,7 +43,6 @@ zenml example pull standard_interfaces
 cd zenml_examples/standard_interfaces
 
 # initialize
-git init
```

Comment:
This `git init` is probably present in all of the other examples as well right? If yes I think we should be consistent and remove it everywhere (also docs etc. probably, doesn't have to be in this PR I guess)

Reply:
This exists as a [ticket](https://zenml.atlassian.net/browse/ENG-293), just applied the boy scout principle and removed it in the files I touched.

Reply:
Alright 👍